projects/angular-auth-oidc-client/src/lib/auth.module.ts

```diff
@@ -62,7 +62,6 @@ import { TokenValidationService } from './validation/token-validation.service';
 export interface PassedInitialConfig {
   config?: OpenIdConfiguration | OpenIdConfiguration[];
   loader?: Provider;
-  storage?: any;
```

Comment:
Please double-check if this is documented anywhere. If we remove this property, we also remove it from the docs.

Reply:
we have it only on [here ](https://github.com/damienbod/angular-auth-oidc-client/blob/89913ff47194ba1ce29af5e29059de15cfdb38d3/docs/site/angular-auth-oidc-client/docs/migrations/v13-to-v14.md#providing-a-custom-storage-service)